src/osgPlugins/dae/daeRGeometry.cpp

```diff
@@ -456,10 +440,10 @@ osg::Geode *daeReader::processGeometry(domGeometry *pDomGeometry)
 
 
 template< typename T >
-void daeReader::processSinglePPrimitive(osg::Geode* geode,
+void daeReader::processSinglePPrimitive(osg::Group* geometryGroup,
     const domMesh* pDomMesh, const T* group, SourceMap& sources, GLenum mode)
 {
-    osg::ref_ptr<osg::Geometry> geometry = new osg::Geometry();
```

Comment:
This is the only case where osg::ref_ptr<osg::Geometry> is used, other templates use just pointers. Was there a specific reason for this?